src/main/java/cookie/Cookie.java

```diff
@@ -0,0 +1,27 @@
+package cookie;
+
+import model.User;
+
+public class Cookie {
```

Comment:
```suggestion
public class Session {
```
클라이언트에 저장된 값은 쿠키, 서버에서 사용하는 것은 세션입니다.